### DISCUSSION TITLE
每一章的标题前的空白有点大

### DISCUSSION THREAD

**Discussion by BystanderJ** [whutug/whu-thesis#133] [Q&A]
每一章的标题前面的空白有点大，如何消除呢？

**Comment by tanukihee** [comment]
https://github.com/whutug/whu-thesis/blob/ef9de6bf741e946f939d929e886e1e59843d7ed7/whu-thesis.cls#L314 这里

**Reply by BystanderJ** [reply]
感谢大佬解答！
